articles/avoid-fix-flaky-tests.md

```diff
@@ -0,0 +1,79 @@
+---
+authorName: Adrian Herzog
+authorGithubUsername: adiherzog
+issue: 42
+title: How to deal with flaky system tests
+---
+
+# {{page.title}}
+
+This article is not about whether UI / system / end to end tests should be automated or not. There's enough material and controversy about that in the developer community and it could easily fill a separate article, if not a whole book. Instead I will fokus on strategies to keep these tests as stable as possible.
+
+## The problem with flakiness
+
+A test that changes from red to green or green to red without any code changes is considered flaky. Such tests are a burden for several reasons. They might block you from merging to the mainline or to release a new version of your product. They might be a symptom of an unstable test or of a problem in the product. In the end, dealing with all these issues eats up valueable time. That's why we want to avoid flaky tests as much as possible.
+
+## Increase your chances to succeed
+
+### Avoid broken windows
+
+As soon as one test fails, insist on fixing it. Otherwise people might start ignoring failures and you'll end up in a [broken window](https://en.wikipedia.org/wiki/Broken_windows_theory) situation. There's a root cause to every flaky test. Either it's the test itself that needs improvement or it's a bug in the software.
+
+### Control as much as possible
```

Comment:
Die ganze Liste unten finde ich noch etwas holprig. Es liegt wohl daran, dass es Lösungen und Fallback Lösungen präsentiert werden. Zum Beispiel: "use simulators" und dann nachher "other systems you rely on". Die gäbe es ja gar nciht, wenn ausschliesslich Simulatoren verwendet werden. Konkret habe ich aber keinen Vorschlag, wie man das klarer ausdrücken könnte.

Reply:
I rephrased some of the bullet points and also reordered them.